client/src/components/JobInformation/JobInformation.vue

```diff
@@ -5,12 +5,30 @@
             <tbody>
                 <tr v-if="job && job.tool_id">
                     <td>Galaxy Tool ID:</td>
-                    <td id="galaxy-tool-id">{{ job.tool_id }}</td>
+                    <td id="galaxy-tool-id">
+                        {{ job.tool_id }} (<a :href="'/root?tool_id=' + job.tool_id" target="_top">run</a>)
+                    </td>
                 </tr>
                 <tr v-if="job && job.tool_version">
                     <td>Galaxy Tool Version:</td>
                     <td id="galaxy-tool-version">{{ job.tool_version }}</td>
                 </tr>
+                <tr v-if="job">
+                    <td>Created by workflow?</td>
```

Comment:
We use this component also when displaying workflow invocations, so this would be redundant there. I think there's some logic for making this customizable with slots, I think that'd be a good use for parameterizing whether to display this or not.

Reply:
Or a `v-if` and a new prop would work too.

Reply:
Something like this @mvdbeek: https://github.com/galaxyproject/galaxy/pull/11244/commits/3d2a1fb3252ffd0067b49d26a8a92b5ad681dfd4 ?

It is not working currently, unfortunately.

Reply:
Yes, but you also need to set the prop. In client/src/components/WorkflowInvocationState/JobStep.vue  you don't need to create a new prop, you only need to pass `:showWorkflowInfo="false"` in <job-information/>

Reply:
Got it, thanks.